pkg/maps/nat/nat.go

```diff
@@ -98,6 +102,73 @@ func NewMap(name string, family IPFamily, entries int) *Map {
 	}
 }
 
+func startingChunkSize(maxEntries int) int {
+	bucketSize := math.Sqrt(float64(maxEntries * 2))
```

Comment:
Can you spell out a bit more about this math here? I'm curious about how we obtain this bucketSize.

Reply:
I took this from a discussion that @antonipp and @christarazi had regarding a similar usage of batched iteration.

I _think_ this is this approximation of the [Birthday Paradox ](https://en.wikipedia.org/wiki/Birthday_problem#Square_approximation) where we set P(n) = 1 😄